tests/manual-test-cases/Group19-VIC-Product/19-5-Configuration-1-VC6.5-ESX6.5-VIC-OVA-Harbor-Secured-SelfSignedCACert.md

```diff
@@ -0,0 +1,100 @@
+4. Tag the image/application to push it to Harbor private registry using local docker in the client machine
+5. Push the image/application from local to Harbor Private registry
+6. From VCH host, Pull the image from Harbor registry
+7. Run the application using docker run command
+8. Stop, Start, attach, kill, rm and rmi
+
+###Test Pos002 Developer Operations:
+1. Reuse the VCH created in test Pos001
+2. Do docker login to harbor using an user with developer role
+3. Pull an image/application into the local docker registry using local docker in the client machine
+4. Tag the image/application to push it to Harbor private registry using local docker in the client machine
+5. Push the image/application from local to Harbor Private registry
+6. From VCH host, Pull the image from Harbor registry
+7. Run the application using docker run command
+8. Stop, Start, attach, kill, rm and rmi
+
+###Test Neg001 Developer Operations:
+1. Reuse the VCH created in test Pos001
+2. Do docker login to harbor using an user with guest role
+3. Pull an image/application into the local docker registry using local docker in the client machine
+4. Tag the image/application to push it to Harbor private registry using local docker in the client machine
+5. Push the image/application from local to Harbor Private registry
+
+###Test Pos003 Two VCH With One Harbor:
+1. Create a VCH2 with Harbor private registry as an option (using --registry-ca <harbor-ip>:80)
+2. Using VCH1, Do docker login to harbor registry using a developer user
+3. Using VCH2, Do docker login to harbor registry using another developer user
+4. Using VCH1, Pull an image/application into the local docker registry using local docker in the client machine
+5. Using VCH1, Tag the image/application to push it to Harbor private registry using local docker in the client machine
+6. Push the image/application from local to Harbor Private registry
+7. From VCH host, Pull the image from Harbor registry
+8. Run the application using docker run command
+9. Stop, Start, attach, kill, rm and rmi
+
+###Test Pos004 Three Client Machines With One Harbor:
+1. Create a VCH with Harbor private registry as an option (using --registry-ca <harbor-ip>:80) from any client machine
+2. Using different machines, connect to the same VCH (in parallel)
+3. Do docker login to harbor using different users (in parallel)
+4. Pull an image/application into the local docker registry using local docker in the client machine
+5. Tag the image/application to push it to Harbor private registry using local docker in the client machine
+6. Push the image/application from local to Harbor Private registry
+7. From VCH, Pull same/different images using different users from Harbor registry
+8. Run the applications using different users from Harbor registry (in parallel)
+9. Stop, Start, attach, kill, rm and rmi using different users from Harbor registry (in parallel)
+
+#Expected Outcome:
```

Comment:
Nit, but could you put the expected outcome of each test with each test case step description?